docs/release-notes.rst

```diff
@@ -1,6 +1,46 @@
 Release Notes
 =============
 
+Restraint 0.2.1
+---------------
+
+Upgrades:
+~~~~~~~~~
+* | Provide ease of use with Restraint commands
+  | When running commands outside of jobs on the local host,
+    some `restraint` commands require manually setting up
+    environment variables or constructing long URLs
+    before running. This can be issue if you are trying
+    to extend the watchdog in a timely fashion.
+    A new option was added which requires the
+    argument `--port <restraint-port-number>`.
+    Commands affected are rstrnt-report-log,
+    rstrnt-report-result, rstrnt-abort, and
+    rstrnt-watchdog.
+* | :bug:`1802610`: Document how to remove RHTS from Jobs
+  | Added new section :ref:`rm_rhts_guide` detailing
+    substitutes for RHTS scripts, environment variables,
+    and testinfo.desc file and associated variables.
+
+Bug Fixes:
+~~~~~~~~~~
+* | Revert process stdin behavior
+  |  Daniel to provide input.
+* | Revert logging in server_io_callback
+  |  Daniel to provide input.
```

Comment:
This relates to the replaced call to g_error too. Logging in task and server handler are restored back to 0.1.45 release. Plus, those bugs for improper use of buffer.